cypress/e2e/auth_spec/redirect.cy.ts

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+
+import { cy, describe, it } from "local-cypress";
+
+describe("redirect", () => {
+  it("Check if login redirects to the right url", () => {
+    cy.log("Logging in the user staffdev:Coronasafe@123");
+
+    cy.visit("/resource/board");
+    cy.get("input[id='username']").type("staffdev");
+    cy.get("input[id='password']").type("Coronasafe@123");
+    cy.get("button").contains("Login").click();
+
+    cy.get("p").contains("Sign Out").should("exist");
+    cy.url().should("include", "/resource/board");
+  });
+
+  it("Check if the redirect param works", () => {
+    cy.log("Logging in the user staffdev:Coronasafe@123");
+
+    cy.visit("login?redirect=http://localhost:4000/resource/board");
+    cy.get("input[id='username']").type("staffdev");
```

Comment:
once you create the reusable login, you don't need to keep on specifying the user credentials